packages/components/src/components/Dropdown/Dropdown.stories.tsx

```diff
@@ -1,6 +1,6 @@
 import React from 'react';
-import styled from 'styled-components';
```

Comment:
I would also remove [those 2 rules](https://github.com/trezor/trezor-suite/blob/c9c7d7db09e51a8cc50891b6fc383d1c3aacc6f9/packages/components/tsconfig.json#L8) as a part of Part 2? @tomasklim Removing the `components` package from the blacklist would be the next step, many more rules come in actions if that's done.

Reply:
I agree